documentation/development-docs/vm-for-partners.md

```diff
@@ -0,0 +1,126 @@
+If you need an Azure Virtual Machine with all the [prerequisites](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/azure-powershell-developer-guide.md#prerequisites) installed and ready to build and test Powershell Azure - follow the steps below. 
+
+# Copy the PowerShell Azure VHD blob to your subscription and create a VM form it.
+
+1. Make sure that you have the version 4.5.0 or above of the AzureRM.Compute PowerShell module. Run the following command to install it. This version has the "easy VM create" cmdlet that we are going to use in the script below to create a VM.
+    ```PowerShell
+    Install-Module AzureRM.Compute --MinimumVersion  4.5.0
+    ```
+2. Request a VHD SAS URI form us and assign it to the variable.
+    ```PowerShell
+    $VhdSasUri = "https://aaa.blob.core.windows.net/bbb/ccc?some-signature"
+    ```
+3. Create some variables..
+   ```PowerShell
+    $SubscriptionId = <subscription-id>
+    $ResourceGroupName = <resource-group-name>
+    $Location = <location>
```

Comment:
Looks like this is not a trivial task. I tried a couple of cmdlets out (Get-AzureRmVMSize and Get-AzureRmResourceProvider) to check supported location - even they confirmed the location was supported - the actual VM creation cmdlet failed.

Reply:
Can you still include the cmdlet to check supported locations? This will at least narrow it down for people (and maybe include a note that westus often fails because of vm limits on a subscription).